rosa/rosa-wrapper.py

```diff
@@ -321,6 +324,20 @@ def main():
             exit(1)
         exit(index_result)
 
+    if args.only_delete_clusters:
+        try:
+            logging.info('Reading cluster name seed from %s' % args.path)
+            cluster_name_seed_file = open(args.path + '/cluster_name_seed')
+            cluster_name_seed = cluster_name_seed_file.read().replace("\n","")
+            logging.info('Found cluster name seed as: %s' % cluster_name_seed)
+        except Exception as err:
+            logging.error(err)
+            logging.error('Failed to read %s/cluster_name_seed file' % args.path)
+            exit(1)
+        rosa_cmnd = _verify_cmnd(args.rosa_cli,args.path)
+        _cleanup_clusters(rosa_cmnd,cluster_name_seed)
+        exit(0)
```

Comment:
Ideally, we should add logic to cleanup_clusters that has it return 0/1 and then exit with that return code.